manim/mobject/mobject.py

```diff
@@ -1308,44 +1373,35 @@ def apply_points_function_about_point(
             mob.points += about_point
         return self
 
-    @deprecated(
-        since="v0.11.0",
-        until="v0.12.0",
-        replacement="rotate",
-    )
-    def rotate_in_place(self, angle, axis=OUT):
+    def rotate_in_place(self: MOS, angle: float, axis: np.ndarray = OUT) -> MOS:
         # redundant with default behavior of rotate now.
         return self.rotate(angle, axis=axis)
 
-    @deprecated(
-        since="v0.11.0",
-        until="v0.12.0",
-        replacement="scale",
-    )
-    def scale_in_place(self, scale_factor, **kwargs):
+    def scale_in_place(
+        self: MOS, scale_factor: float, **kwargs: np.ndarray | None
+    ) -> MOS:
         # Redundant with default behavior of scale now.
         return self.scale(scale_factor, **kwargs)
 
-    @deprecated(
-        since="v0.11.0",
-        until="v0.12.0",
-        replacement="scale",
-    )
-    def scale_about_point(self, scale_factor, point):
+    def scale_about_point(self: MOS, scale_factor: float, point: np.ndarray) -> MOS:
```

Comment:
Why did you remove the depreciation?

Reply:
Weird, I definitely didn't do that! Maybe an autoformatter gone wrong? Good spot though, I'll add it back.